python/src/KrigingAlgorithm_doc.i.in

```diff
@@ -153,9 +153,26 @@ Create the Kriging algorithm with the optimizer option:
 
 Parameters
 ----------
-problem : :class:`~openturns.Interval`
-    The bounds used for numerical optimization of the likelihood."
+bounds : :class:`~openturns.Interval`
+    The bounds used for numerical optimization of the likelihood.
+
+Notes
+-----
+Parameters involved by this method are:
+
+ - Scale parameters,
+ - Amplitude parameters if output dimension is greater than one or
+   analytical sigma disabled,
+ - Additional parameters.
+
+Default bounds in the library are :math:`[100]^d` with `d` the size
```

Comment:
same remarks here. BTW, it would be a good thing to be able to avoid these duplicates. Maybe a link to GeneralLinearModelAlgorithm::setBounds() would be better.